dvc/machine/backend/terraform.py

```diff
@@ -12,9 +14,42 @@
     from dvc.types import StrPath
 
 
-class TerraformBackend(terraform.TerraformBackend, BaseMachineBackend):
+class TerraformBackend(BaseMachineBackend):
```

Comment:
@pmrowla, any reason why you don't make the `MachineManager`/`Repo.machine` lazy like we do in `scm`/`index`, etc.?

https://github.com/iterative/dvc/blob/2abe515391cdbfa1898747c4a4a8d997fb42962f/dvc/repo/__init__.py#L264-L265

Reply:
No reason, I'll add that change

Reply:
You may be able to rollback some lazy imports from here with that.